src/solvers/odepack.py

```diff
@@ -186,7 +186,7 @@ def integrate_start(self, t, y):
         """
         Helper program for the initialization of LSODAR
         """
-        #print ' We have rkstarter {} and rkstarter_active {}'.format(self.rkstarter, self._rkstarter_active)
+        #print('We have rkstarter {} and rkstarter_active {}'.format(self.rkstarter, self._rkstarter_active))
```

Comment:
Should we remove this line since it is a comment?

Reply:
Removed.